maven-resolver-provider/src/test/java/org/apache/maven/repository/internal/ArtifactDescriptorReaderDelegateTest.java

```diff
@@ -0,0 +1,79 @@
+package org.apache.maven.repository.internal;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.maven.model.Dependency;
+import org.apache.maven.model.DependencyManagement;
+import org.apache.maven.model.Model;
+import org.eclipse.aether.artifact.DefaultArtifact;
+import org.eclipse.aether.resolution.ArtifactDescriptorRequest;
+import org.eclipse.aether.resolution.ArtifactDescriptorResult;
+
+public class ArtifactDescriptorReaderDelegateTest extends AbstractRepositoryTestCase{
+    public void testMNG7509()
```

Comment:
Please format this file according to our rules

Reply:
Hi @michael-o , I'd like to double confirm to make sure my understanding is correct. Since I create this testcase refer to an existing one org.apache.maven.repository.internal.DefaultArtifactDescriptorReaderTest, I'm not sure whether the format doesn't follow your rules means the class and method name is not quite meaningful.
If it is, I will change it. If not, please let me know what should I do.
Thanks.

Reply:
https://maven.apache.org/developers/conventions/code.html

Reply:
Reformat done